core/templates/components/forms/schema-based-editors/schema-based-custom-editor.component.spec.ts

```diff
@@ -93,4 +120,23 @@ describe('Schema Based Custom Editor Component', () => {
 
     expect(component.localValue).toBe('true');
   });
+
+  it(
+    'should register validator and call it when the form validation changes',
+    fakeAsync(() => {
+      component.schema = { obj_type: 'UnicodeString', type: 'custom' };
+      fixture.detectChanges();
+      flush();
+      const spy = jasmine.createSpy('validator onchange spy');
+      let mockEmitter = new EventEmitter();
+      component.registerOnValidatorChange(spy);
+      spyOnProperty(
+        component.hybridForm, 'statusChanges').and.returnValue(mockEmitter);
+      component.ngAfterViewInit();
+      component.validate(new FormControl());
+      mockEmitter.emit();
+      tick();
+      expect(spy).toHaveBeenCalled();
+    }
+    ));
```

Comment:
Deindent by 2

Reply:
I moved the brackets to the previous line